compiler/testData/codegen/box/annotations/parameters.kt

```diff
@@ -1,3 +1,5 @@
+// DONT_TARGET_EXACT_BACKEND: WASM
```

Comment:
Why you don't use IGNORE_BACKEND instead?

Reply:
Primary reason is to not waste time running tests that are known to fail.